.github/workflows/ci.yml

```diff
@@ -44,6 +44,12 @@ jobs:
       - name: Launch pytest tests
         run: coverage run --branch -m pytest -v
 
+      - name: Launch pytest tests not requiring INGInious modules
+        run: |
+          virtualenv env
+          env/bin/pip3 install jinja2 pytest coverage
+          env/bin/coverage run --branch -m pytest -v utils
```

Comment:
Running this on my machine fails if I have the ``inginious`` folder next to the `utils` folder, which is the default tree. `coverage` is able to load the `inginious` modules as the folder is located in the running directory. Then running `python3 utils/task_tester/inginious-submission-anonymizer` in the test fails because this script won't find the `inginious` modules in its own running directory...

Is this behaviour expected ? What are the exact conditions for running those tests ?

Reply:
That is strange, no test should fail but some must be skipped. This run of the CI shows the expected behavior https://github.com/UCL-INGI/INGInious/actions/runs/3384962440/jobs/5622553767#step:7:92, https://github.com/UCL-INGI/INGInious/actions/runs/3384962440/jobs/5622553767#step:8:46.

The first test verifies the behavior of the anonymizer when INGInious is not locally installed (which is the case for most of the task developers). As the modules are installed within the CI, we use a virtualenv to run this test in a clean environment.

The two last tests expect the INGinious modules to be installed. The fact that the tests are executed within the INGInious directory should not impact the tests.

Reply:
The tests pass with a clean environment. There should be some junk left in my dev folder.